openapi.yaml

```diff
@@ -0,0 +1,183 @@
+openapi: 3.1.1
+info:
+  title: Bundestag API
+  description: API for retrieving information about members of the German Bundestag.
+  version: 1.0.0
+servers:
+  - url: https://bundestag-api.kyzrlabs.cloud
+paths:
+  /politicians:
+    get:
+      summary: Retrieve a list of all members of the German Bundestag.
+      responses:
+        '200':
+          description: Successful response with the list of members.
+  /politicians/{id}:
+    get:
```

Comment:
Need to add Accept Header  „image/webp“ producing byte return 

https://swagger.io/docs/specification/v3_0/describing-parameters/